### PR TITLE
IF-302: Alert notifications

### PR DESCRIPTION
**Pull request by jsylupp**

### CODE REVIEW

**Comment by mattmikulina** [comment]
Task linked: [IF-302 Show and edit Notifications](https://app.clickup.com/t/2177764/IF-302)

File: fusionfrontend/src/app/notifications/content/notifications-sub-header/notifications-sub-header.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
```
Comment (seemueller) [inline fusionfrontend/src/app/notifications/content/notifications-sub-header/notifications-sub-header.component.spec.ts:1]: licence text ist missing

File: fusionfrontend/src/app/notifications/pages/notifications-page/notifications-page.component.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+//  @import '../base/base-list/base-list.component.scss';
```
Comment (seemueller) [inline fusionfrontend/src/app/notifications/pages/notifications-page/notifications-page.component.scss:16]: remove unused code

File: fusionfrontend/src/app/notifications/pages/notifications-page/notifications-page.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
```
Comment (seemueller) [inline fusionfrontend/src/app/notifications/pages/notifications-page/notifications-page.component.spec.ts:1]: missing licence text

**Review by seemueller — CHANGES_REQUESTED** [review]

File: fusionfrontend/src/app/notifications/content/notifications-list/notifications-list.component.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+<p-overlayPanel #searchpanel [showCloseIcon]="true" class="assetSearchPanel">
+  <ng-template pTemplate>
+    <div class="panel-heading">Search</div>
+    <span class="p-input-icon-left">
+            <fa-icon [icon]="faSearch" class="fa-fw" ></fa-icon>
+            <input type="text" pInputText [(ngModel)]="searchText" placeholder="Name" class="search-input">
+        </span>
+    <br>
+    <div class="footer">
+      <button type="button" class="if-button button-primary button-modal" style="font-size: 13px; height: 30px; margin: 5px 0 0 0;"
```
[the remark this answers is not in the document]
Comment (jsylupp) [reply fusionfrontend/src/app/notifications/content/notifications-list/notifications-list.component.html:61]: Will be solved in other story anyway

**Pull request merged**